src/components/Layout/NavigationLayout/GenreNavigation/GenreNavigation.tsx

```diff
@@ -1,11 +1,51 @@
+import { GENRE_FILTER } from '../../../../constants/constants';
+import { moviesActions } from '../../../../redux/moviesSlice';
+import { useAppDispatch, useAppSelector } from '../../../../redux/store';
+
 import './GenreNavigation.scss';
 
 const GenreNavigation = (): JSX.Element => {
-  const genres = ['All', 'documentary', 'comedy', 'horror', 'crime'];
+  const filters = useAppSelector((state) => state.movies.filters);
+  const dispatch = useAppDispatch();
+  const allFilters = GENRE_FILTER[0].id;
 
-  const genresTemplate = genres.map((genre) => <div key={genre}>{genre}</div>);
+  let genres = [];
 
-  return <div className="mwGenresContainer">{genresTemplate}</div>;
+  const initMenuOptions = (): JSX.Element => {
+    GENRE_FILTER.forEach((option) => {
```

Comment:
I'd suggest to remove the function and use `GENRE_FILTER.map` directly in the return statement to render buttons - that way readability would be much better.

Reply:
as for me with forEach it is more clearer. perhaps will change it.